Components/SettingsBar/SettingsBarItem.tsx

```diff
@@ -0,0 +1,15 @@
+import { MouseEventHandler } from "react";
+
+const SettingsBarItem = ( { itemName, iconName, isActive, onClick } : { itemName:string, iconName:string, isActive:boolean, onClick:MouseEventHandler} ) => {
+  return (
+    <div 
+      className={"flex items-center hover:bg-zinc-100 py-2 px-2 cursor-pointer rounded-md text-sm".concat(isActive?" text-zinc-900 bg-zinc-100":" text-zinc-700 bg-white")}
```

Comment:
Do not use concat here, either use `+` or `${ }`. Also, set the default text color given in the issue.

Reply:
@PiyushKalyanpy Sir, working on it 🚀🚀🚀